src/styles/blocks/product.scss

```
@@ -0,0 +1,68 @@
+.product {
+  width: 200px;
+  padding: 16px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 16px auto 40px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $text-color-dark;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    color: $text-color-light;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price {
+    margin-top: 6px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__button {
+    height: 40px;
+    margin-top: 16px;
+    border-radius: 5px;
+    background-color: #00acdc;
+
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 40px;
+    text-align: center;
+    color: #fff;
+
+    &:hover {
+      background-color: #fff;
+      border: 1px solid #00acdc;
```

Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover
```suggestion
      border-color: #00acdc;
```